doc/developers/contributing.rst

```diff
@@ -1190,3 +1190,32 @@ that implement common linear model patterns.
 
 The :mod:`sklearn.utils.multiclass` module contains useful functions
 for working with multiclass and multilabel problems.
+
+Estimator Tags
+--------------
+Scikit-learn introduced estimator tags in version 0.19.
+These are annotations of estimators that allow programmatic inspection of their
+capabilities, such as sparse matrix support, supported output types and
+supported methods.
+The estimator tags are a dictionary returned by the method ``_get_tags()``.
+These tags are used by the common tests and the ``check_estimator`` function to decide
+what tests to run and what input data is appropriate.
```

Comment:
I pinged @mfeurer but I'm not sure if it will actually be helpful so far. Ultimately this would be nice, but for now I'm ok with it closing 4 issues and enabling contrib developers.